src/test/versionbits_tests.cpp

```diff
@@ -32,6 +32,12 @@ class TestConditionChecker : public AbstractThresholdConditionChecker
     int GetStateSinceHeightFor(const CBlockIndex* pindexPrev) const { return AbstractThresholdConditionChecker::GetStateSinceHeightFor(pindexPrev, paramsDummy, cache); }
 };
 
+class TestFixedHeightConditionChecker : public TestConditionChecker
+{
+public:
+    int64_t BeginTime(const Consensus::Params& params) const override { return -1; }
```

Comment:
In commit "Unit tests for always-active versionbits."

Could return ALWAYS_ACTIVE instead of -1 literally here.

Reply:
Done.